exosphere-shared/example-apps/docker-service/application.yml

```diff
@@ -0,0 +1,11 @@
+name: docker-app
+description: An application that relies on dockerhub images
+version: 0.0.1
+
+services:
+  public:
+    local-service:
+      docker_image: exospheredev/local-service
+      location: ./local-service
```

Comment:
Let's follow the principle of least surprise here, and do the simple and straightforward thing that causes the least amount of confusion. I think that is to have only one or the other setting, but not both. Later we can add extra logic, documentation, and testing to allow for both settings being present at the same time.